### PR TITLE
doc: releases: notes for v14.2.11 nautilus

### PR DESCRIPTION
**Pull request by theanalyst**
Signed-off-by: Abhishek Lekshmanan <abhishek@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by theanalyst** [comment]
jenkins render docs

File: doc/releases/nautilus.rst [hunk 1]
```diff
@@ -1,3 +1,104 @@
...
+  results on the cluster, which could fill a nearly-full cluster.
+  They have been replaced by a tool, currently considered
+  experimental, ``rgw-orphan-list``.
+
+* Now when noscrub and/or nodeep-scrub flags are set globally or per pool,
+  scheduled scrubs of the type disabled will be aborted. All user initiated
+  scrubs are NOT interrupted.
+
+* Fixed a ceph-osd crash in committed osd maps when there is a failure to encode
+  the first incremental map. `issue#46443 <https://github.com/ceph/ceph/pull/46443>`_
```
Comment (neha-ojha) [inline doc/releases/nautilus.rst:24]:
```suggestion
* Fixed a ceph-osd crash in _committed_osd_maps when there is a failure to encode the first message in the incremental osdmap. `issue#46443 <https://github.com/ceph/ceph/pull/46443>`_
```

Comment (theanalyst) [reply doc/releases/nautilus.rst:24]: ack

File: doc/releases/nautilus.rst [hunk 2]
```diff
@@ -1,3 +1,104 @@
+v14.2.11 Nautilus
+=================
+
+This is the eleventh release in the Nautilus series. This release brings a
```
Comment (batrick) [inline doc/releases/nautilus.rst:4]: technically it's the twelfth since we started at 0 :)

Comment (theanalyst) [reply doc/releases/nautilus.rst]: made it the eleventh backport release :)

**Pull request merged**